Added Maestro rollback steps into  FAQ (#6775)
dotnet/arcade: Documentation/Projects/M2MRenaming/Master-to-Main-renaming-guide.md

GitHub links are automatically redirected. For example https://github.com/dotnet/xharness/blob/master/README.md will still work after the rename and will point to https://github.com/dotnet/xharness/blob/main/README.md.

GitHub raw links are automatically redirected. For example link https://raw.githubusercontent.com/dotnet/xharness/master/README.md still works even after rename and is equivalent to link https://raw.githubusercontent.com/dotnet/xharness/main/README.md.

### How to revert Maestro migration?
Two update scripts are generated. There are 3 scenarios:

> * Only the script "disable-subscriptions-in-maestro.ps1" was executed. To roll-back edit this script and replace argument -d with -e.

Reviewer: I suggest adding an example of the full command. It's common that the person who needs a rollback is not the person who set things up initially, so having the full command saves them the time of reading other sections.

> * Only the script "disable-subscriptions-in-maestro.ps1" was executed. To roll-back edit this script and replace argument -d with -e.

Reviewer 2: ```suggestion
* In case the `disable-subscriptions-in-maestro.ps1` script was executed **only**, to roll back, edit this script and replace argument `-d` with `-e`.
```

> * Both scripts were executed. To roll-back you need to generate roll back scripts using another call of "./disable-subscriptions-prepare-migration-script.ps1 [repo name] master main". Then execute generated update script "./rename-branch-in-maestro.ps1" and all changes will be reverted.

Reviewer 2: ```suggestion
* When both scripts were executed, you need to generate rollback scripts using the second script:
    `./disable-subscriptions-prepare-migration-script.ps1 [repo name] master main`.
    Then execute generated update script `./rename-branch-in-maestro.ps1` and all changes will be reverted.
```

> * If there is any other error during execution of update scripts please contact us.

Reviewer 2: ```suggestion
* Reach out to us in case of any questions or issues with these scripts.
```
